### PR TITLE
chore: Specify culture invariant

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Details

Specify CultureInfo.InvariantCulture when rendering strings.

##### Motivation

This is part of the GlobalReadiness work for the compliance feature. Only 1 change (called out in the code) is likely to have any real impact to the shipping code.

#### Pull request checklist

- [ ] Addresses an existing issue: #0000

### CODE REVIEW

File: src/Core/Bases/A11yProperty.cs [hunk 1]
```diff
@@ -147,7 +148,7 @@ private string GetBoundingRectangleText()
             if ((double)arr[2] < 0 || (double)arr[3] < 0)
             {
                 // the 3rd and 4th values in array are negative value, we need to show value in different format like l,t,w,h
-                text = string.Format("[l={0},t={1},w={2},h={3}]", arr[0], arr[1], arr[2], arr[3]);
+                text = string.Format(CultureInfo.InvariantCulture, "[l={0},t={1},w={2},h={3}]", arr[0], arr[1], arr[2], arr[3]);
```
Comment (DaveTryon) [inline src/Core/Bases/A11yProperty.cs:151]: This is the ship-impacting change. By default, the numbers in the string should pick up culture-specific separators. With CultureInfo.InvariantCulture, all separators should be skipped.

File: src/Core/Types/PatternType.cs [hunk 2]
```diff
@@ -80,7 +80,7 @@ protected override string GetNameInProperFormat(string name, int id)
 
             sb.Replace("UIA_", "");
             sb.Replace("Id", "");
-            //sb.AppendFormat("({0})", id); // skip Id for now
+            // sb.Append(Invariant($" ({id})")); // skip id for now
```
Comment (DaveTryon) [inline src/Core/Types/PatternType.cs:83]: Updating the commented out code to follow the same pattern used in other types.

File: src/Core/Types/PlatformPropertyType.cs [hunk 3]
```diff
@@ -51,7 +51,7 @@ protected override string GetNameInProperFormat(string name, int id)
 
             sb.Replace("Platform_", "");
             sb.Replace("PropertyId", "");
-            //sb.AppendFormat("({0})", id);
+            // sb.Append(Invariant($" ({id})"));
```
Comment (DaveTryon) [inline src/Core/Types/PlatformPropertyType.cs:54]: Updating the commented out code to follow the same pattern used in other types.

File: src/Desktop/Styles/AnimationStyle.cs [hunk 4]
```diff
@@ -61,6 +61,7 @@ protected override string GetNameInProperFormat(string name, int id)
 
             sb.Replace(Prefix, "");
             sb.Append(Invariant($" ({id})"));
+
```
Comment (DaveTryon) [inline src/Desktop/Styles/AnimationStyle.cs:64]: Newline added to match the formatting used in most styles.

File: src/Desktop/Styles/BulletStyle.cs [hunk 5]
```diff
@@ -60,6 +60,7 @@ protected override string GetNameInProperFormat(string name, int id)
 
             sb.Replace(Prefix, "");
             sb.Append(Invariant($" ({id})"));
+
```
Comment (DaveTryon) [inline src/Desktop/Styles/BulletStyle.cs:63]: Newline added to match the formatting used in most styles.

File: src/Desktop/Styles/CapStyle.cs [hunk 6]
```diff
@@ -61,6 +61,7 @@ protected override string GetNameInProperFormat(string name, int id)
 
             sb.Replace(Prefix, "");
             sb.Append(Invariant($" ({id})"));
+
```
Comment (DaveTryon) [inline src/Desktop/Styles/CapStyle.cs:64]: Newline added to match the formatting used in most styles.

File: src/Core/Bases/A11yProperty.cs (hunk 1 above)
Comment (RobGallo) [reply src/Core/Bases/A11yProperty.cs:151]: Does it then make sense to use the current culture?

Comment (DaveTryon) [reply src/Core/Bases/A11yProperty.cs:151]: Upon further investigation, separators are not impacted by the current culture, but negative signs definitely are. This is [documented](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.numberformatinfo.negativesign?view=net-5.0) and I confirmed it via a small test app. For a data file format, we should be using culture-agnostic formats both when reading and when writing, so a file created in culture X can be opened in culture Y.

**Review by RobGallo — APPROVED** [review]

**Pull request merged**